package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "main": "index.js",
   "license": "GPL-3.0",
-  "version": "1.22.0",
+  "version": "1.22.4",
```

Comment:
Is this the correct version to end up in develop?

Reply:
I accidentally published a few versions with bugs in them. You can't undo a publish, so that's why.